backend/modules/pdf_processing/router.py

```
@@ -1 +1,287 @@
+    
+    # Start processing in the background
+    async def run_parsing():
+        """Background task for PDF processing"""
+        try:
+            pipeline = get_pipeline(db, current_user)
+            await pipeline.process_full_with_progress(file, session_id, context_files)
+        except HTTPException as e:
+            debug_log(f"[ENDPOINT] HTTPException in background task: {e.status_code} - {e.detail}")
+            if session_id:
+                progress_manager.error_processing(session_id, f"{e.detail}")
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)